manifests/repo.pp

```diff
@@ -33,6 +33,14 @@
         version => $version,
       }
     }
+    'Debian': {
+      if ! defined( '::apt' ) {
```

Comment:
I fear it wouldn't parse if there's no apt module available, but I'll proceed as you prefer